fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/GroupByAdapter.java

```diff
@@ -13,18 +13,33 @@
     // The select statement we are adapting
     private final Select select;
 
+    // The group by clause for the select we are building
+    private final GroupByClause groupByClause;
+
     /**
      * Protected constructor for modeling the GROUP BY part of a SELECT statement.
-     * 
+     *
      * @param select
      */
-    protected GroupByAdapter(Select select) {
+    protected GroupByAdapter(Select select, GroupByClause gb, String... expressions) {
```

Comment:
add gb and expression @param annotations

Reply:
ok